src/app/services/data-table.service.ts

```diff
@@ -117,7 +123,9 @@ export class DataTableService {
   }
 
   set dataTable(value: DataTable) {
-    this._dataTable = value;
+    if (value instanceof DataTable) {
```

Comment:
In what situations is something passed that is not an instance of `DataTable`?

Reply:
seems to be related to zone.js regressions, but not entirely sure, no point in the our code that assigns values other than DataTable, but while chaining promises, boolean values somehow got assigned to data table